### PR TITLE
fix(jerseyExtension):  flaky test ExceptionMappersIntegrationTest

### LINKED ISSUE #2141: ExceptionMappersIntegrationTest fails when building from newest version
# Bug Report

## Describe the Bug
i cloned the repository and ran the gradlew.bat clean build and got 1 failed test. 

ExceptionMappersIntegrationTest
shouldListJakartaValidationMessageInResponseBody()

I have attached the full errormessage.

[Exception.txt](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/files/9870974/Exception.txt)

### PR DESCRIPTION
**Pull request by wolf4ood**
## What this PR changes/adds

Fix flaky test ExceptionMappersIntegrationTest

## Why it does that

Tests were failing on a locale which was not `en`

## Linked Issue(s)

Closes #2141 

## Checklist

- [ ] added appropriate tests?
- [x] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] added/updated relevant documentation?
- [x] assigned appropriate label? (exclude from changelog with label `no-changelog`)
- [x] formatted title correctly? (_take a look at the [CONTRIBUTING](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CONTRIBUTING.md#submit-a-pull-request) and [Etiquette for pull requests](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/pr_etiquette.md) for details_)

### CODE REVIEW

File: extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java [hunk 1]
```diff
@@ -49,18 +52,28 @@
 @ExtendWith(EdcExtension.class)
 public class ExceptionMappersIntegrationTest {
 
+
     private final int port = getFreePort();
     private final Runnable runnable = mock(Runnable.class);
 
+
+    MessageInterpolator interpolator;
```
Comment (ndr-brt) [inline extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java:60]:
```suggestion
    private MessageInterpolator interpolator;
```

File: extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java [hunk 2]
```diff
@@ -49,18 +52,28 @@
...
+
     @BeforeEach
     void setUp(EdcExtension extension) {
+
+
```
Comment (ndr-brt) [inline extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java:65]: Too many empty lines

File: extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java [hunk 3]
```diff
@@ -81,6 +94,8 @@ void shouldListEdcApiExceptionMessageInResponseBody() {
 
     @Test
     void shouldListJakartaValidationMessageInResponseBody() {
+        var positiveTemplate = "{jakarta.validation.constraints.Positive.message}";
+        var notBlankTemplate = "{jakarta.validation.constraints.NotBlank.message}";
```
Comment (ndr-brt) [inline extensions/common/http/jersey-core/src/test/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/ExceptionMappersIntegrationTest.java:98]: They could be constants

**Review by ndr-brt — APPROVED** [review]
Just some nits but neat overall

**Review by jimmarino — APPROVED** [review]

**Pull request merged**